src/test/resources/SampleJson/Progress.json

```diff
@@ -0,0 +1,13 @@
+{
+    "id": 613777,
+    "context_id": 37836,
+    "context_type": "Course",
+    "user_id": null,
+    "tag": "submissions_update",
+    "completion": null,
+    "workflow_state": "queued",
+    "created_at": "2017-04-14T14:32:18Z",
+    "updated_at": "2017-04-14T14:32:18Z",
+    "message": null,
+    "url": "https://k-state.test.instructure.com/api/v1/progress/613777"
```

Comment:
All of the other URLs in our sample JSON files have been changed to `canvas.example.edu` to make things generic and make sure we don't have any valid production URLs in the code base.